GSoC-2025/braingloberegistration automated atlas region selection (Franciszek Mierzejewski).md

```diff
@@ -0,0 +1,79 @@
+| III | Write Up Quantitative Comparison of Methods | 28/05/25 | 01/06/25 | 10  |
+| IV  | Evaluation I – Choose Method | 02/06/25 | 03/06/25 | 5   |
+| V   | Implement Chosen Method | 04/06/25 | 09/07/25 | 40  |
+| VI  | Optimise Code | 10/07/25 | 13/07/25 | 10  |
+| VII | Evaluation II | 14/07/25 | 17/07/25 | 10  |
+| VIII | Conduct Tests (same time as IX) | 18/07/25 | 15/08/25 | 25  |
+| IX  | Rectify Errors Found From Tests | 18/07/25 | 15/08/25 | 10  |
+| X   | Write Up Blog & Documentation | 16/08/25 | 23/08/25 | 20  |
+| XI  | Final Evaluation | 24/08/25 | 31/08/25 | 20  |
+
+I plan to host weekly meetings on Zulip with my mentor discussing current progress and plans. In the case where the timeline is disrupted (e.g. task completion delay), I’ll contact my mentor to discuss the reason behind the delay and the best course of action. It may be in the best interest of NIU to postpone the delayed task to focus on higher priority tasks.
+
+## Personal Statement
+
+### About Me
+
+I’m Franciszek Mierzejewski, a 1<sup>st</sup> year student from the University of Essex studying towards a Bachelor of Science Degree in Computer Science.
+
+### Past Experience
+
+I have worked on an open-source project before – DryPi, a team submission to the 2023 AstroPi competition (hosted by European Space Agency and Raspberry Pi Foundation) that with Artificial Intelligence used images taken from a Rapberry Pi 4 Model 4 to calculate NDVI (Normalised Difference Vegetation Index) values that analysed near-infrared and red light reflectance (a grayscale image) to map to a colourmap to generate into a false-colour image. This false-color image, alongside the International Space Station (ISS) coordinates at the time of the image being taken, was then used to compare with historical open-source data to look at how vegetation density has changed on different continents in the last few decades. It achieved ‘Flight Status’ on the ISS in May 2023. The code for DryPi can be found at: [GitHub - DryPi](https://github.com/FranciszekMierzejewski/AstroPi-Team-DryPi)
+
+![NDVI Usage Image](images/NDVI_usage_example.png)
+
+Figure 1 - Example of NDVI usage
+
+I have had experience with coding in Python (3 years) and NumPy, working with and processing image data. Although I have no experience with ITK or Elastix, I would be learning it as part of my research prior to my project implementation. An example of my experience in machine learning can be taken from a [module](https://www1.essex.ac.uk/modules/Default.aspx?coursecode=CE101&level=4&period=FY&campus=CO) at my university where I applied K-Nearest-Neighbour, Support Vector Machine and Extreme Gradient Boosting (amongst other) algorithms to predict heart disease of patients. The documentation of the project progress can be found here: [C:\\Users\\frane\\OneDrive\\Documents\\CE101_Team_Report_Group_6_Lovelace.pdf](file:///C:\Users\frane\OneDrive\Documents\CE101_Team_Report_Group_6_Lovelace.pdf)
```

Comment:
The link to the pdf isn't visible to anyone else as it's on your onedrive. You will need to generate a sharing link